### PR TITLE
Fix drop menu links

### LINKED ISSUE #64: Fix drop-menu functionality

### PR DESCRIPTION
**Pull request by A-Meri**
reset all status for answers, and quiz data.
added key for answers, and questions inside `reviewAnswers` component.

### CODE REVIEW

File: src/components/ReviewAnswers.js [hunk 1]
```diff
@@ -24,20 +24,21 @@ const ReviewAnswers = ({
         Your score is: {score} / {numOfQuestions}
       </p>
       {questions.map(question => (
-        <Card className="mb-4">
+        <Card key={question.id} className="mb-4">
           <Card.Body>
             <Card.Title>{question.text}</Card.Title>
-            <Card.Text>
+            <div>
```
Comment (bonboh) [inline src/components/ReviewAnswers.js]: I had a look and can see that `Card.Text` gets rendered as a `p`, and MDN says `ul`s should not go within `p`s so good job changing this. However, here the `div` is not doing anything (whereas before `Card.Text` could've had a specific style associated with it which would've made it necessary) so we can remove the `div`

File: src/components/ReviewAnswers.js [hunk 2]
```diff
@@ -24,20 +24,21 @@ const ReviewAnswers = ({
         Your score is: {score} / {numOfQuestions}
       </p>
       {questions.map(question => (
-        <Card className="mb-4">
+        <Card key={question.id} className="mb-4">
```
Comment (bonboh) [inline src/components/ReviewAnswers.js:27]: Great job adding the `key`s to the `map`s, something I missed in my other reviews!

**Review by bonboh — CHANGES_REQUESTED** [review]
Great job! Just one suggestion then we're ready to merge

File: src/components/ReviewAnswers.js (hunk 1 above)
Comment (A-Meri) [reply src/components/ReviewAnswers.js]: I will fix it, thanks 😊

Comment (A-Meri) [reply src/components/ReviewAnswers.js]: yes, about that error, Vicky mentioned to me to fix.

**Review by bonboh — APPROVED** [review]

**Pull request merged**